dmsite/venv/package.json

```diff
@@ -3,15 +3,17 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "antd": "^3.12.3",
+    "antd": "^3.12.4",
     "es": "^0.6.0",
     "filepond": "^3.3.2",
     "prop-types": "^15.6.2",
     "react": "^16.6.3",
     "react-dom": "^16.6.3",
     "react-filepond": "^5.0.0",
+    "react-motion": "^0.5.2",
     "react-router-dom": "^4.3.1",
     "react-scripts": "^2.1.1",
+    "react-strap": "0.0.1",
     "reactstrap": "^6.5.0",
```

Comment:
Is there a difference between 'react-strap' and 'reactstrap'?

Reply:
I think I accidentally installed react-strap but it should be reactstrap. I will get rid of that! Thank you!